edge/pkg/metamanager/metaserver/kubernetes/storage/sqlite/store.go

```diff
@@ -99,6 +100,12 @@ func (s *store) List(ctx context.Context, key string, opts storage.ListOptions,
 		if err != nil {
 			return err
 		}
+
+		labelSet := labels.Set(unstrObj.GetLabels())
+		if !opts.Predicate.Label.Matches(labelSet) {
+			continue
+		}
```

Comment:
Thanks for the fixing, make sense for the `laberselector` bug. And I think we also need to add the field selector here.
We can only add `name` and `namespace` field selector here. Would you like to add them? @chenchunxiu  Or we can add in other PR :) 

Ref: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/selection_predicate.go#L104

Reply:
Done